applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -2069,11 +2069,13 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     {
         NSString *adUnitId = call.arguments[@"ad_unit_id"];
         NSString *adFormatStr = call.arguments[@"ad_format"];
+        id rawIsAdaptiveBannerEnabled = call.arguments[@"is_adaptive_banner_enabled"];
         id rawPlacement = call.arguments[@"placement"];
         id rawCustomData = call.arguments[@"custom_data"];
         id rawExtraParameters = call.arguments[@"extra_parameters"];
         id rawLocalExtraParameters = call.arguments[@"local_extra_parameters"];
         
+        BOOL isAdaptiveBannerEnabled = ( rawIsAdaptiveBannerEnabled != [NSNull null] ) ? ((NSNumber *)rawIsAdaptiveBannerEnabled).boolValue : YES;
```

Comment:
Is it enabled by default on Android too if no value provided?

Reply:
By default, it is enabled.